scrnaseq_hto.Rmd

```diff
@@ -289,8 +492,38 @@ sc
 # Preliminary visualisation of demultiplexed RNA data
 This section of the report provides first insights into your RNA dataset based on a preliminary pre-processing of the RNA data using the standard scRNA-seq workflow. 
 ```{r prelim_analysis, results='hide'}
-sc.all = prelim.analysis(sc=sc.all, mt=params$mt_names, pc_n=10)
-sc = prelim.analysis(sc=sc, mt=params$mt_names, pc_n=10)
```

Comment:
So you skipped the function, and rather write the code in the main script? Is that the intention?

Reply:
Remove the function if it isn't needed any more.

Reply:
Removed function and file